### ISSUE TITLE
Creating a slur on last chord-rest of system will not allow further extension into next system with (ctrl+right)

### ISSUE THREAD

**Issue by worldwideweary** [musescore/MuseScore#19956]
### Issue type

Other type of issue

### Bug description

This is an "oldie but goodie" (for example, 2016: https://musescore.org/en/node/124231)

demonstration of desirable behavior if the slur is enacted at a penultimate chord rest: 

https://github.com/musescore/MuseScore/assets/7139517/f4939d0e-2d11-4975-b730-72c68f9ffac7

Here's the issue demonstrated (slur extends not in anchor points but in offsetting only if last chord rest of system): 

https://github.com/musescore/MuseScore/assets/7139517/7bb153d2-b211-41b8-8469-1ae4bc87cb74

### Steps to reproduce

Doesn't seem to be desirable behavior. Of course a good work around is just to create a range selection from that point on to the desired end point.

### MuseScore Version

4.1 or nightly

### Regression

No.

### Operating system

etc.

**Comment by Jojo-Schmitz** [comment]
Same in Mu3, 2 and 1

**Comment by worldwideweary** [comment]
Yeah, been around for a while. P.S. I tried in the past (3.x)  to merely select the back segment instead of the front when beginning the editing phase, but that didn't change the behavior.
